src/test/java/openfasttrack/importer/tag/TestTagImporter.java

```diff
@@ -213,9 +245,24 @@ private static SpecificationItemId id(final String artifactType, final String na
                 .build();
     }
 
-    private static SpecificationItem item(final SpecificationItemId id)
+    private static SpecificationItem item(final String artifactType, final int lineNumber,
+            final int counter, final SpecificationItemId coveredId)
+    {
+        final SpecificationItemId generatedId = SpecificationItemId.createId(artifactType,
+                generateName(coveredId, lineNumber, counter), 0);
+        return new SpecificationItem.Builder() //
+                .id(generatedId) //
+                .addCoveredId(coveredId) //
+                .build();
+    }
+
+    private static String generateName(final SpecificationItemId coveredId, final int lineNumber,
+            final int counter)
     {
-        return new SpecificationItem.Builder().id(id).build();
```

Comment:
Code duplication in test?

Looks a little bit like: Verify 1 = 1 :stuck_out_tongue_winking_eye: 

Fixed values would be better as expectation. Since filename is a constant that should be no problem.